custom_components/unifi_access/manifest.json

```
@@ -9,7 +9,7 @@
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/imhotep/hass-unifi-access/issues",
   "requirements": [
-    "websocket-client==1.8.0"
+    "websocket-client>=1.8.0"
```

```suggestion
    "websocket-client==1.8.0"
```
I actually prefer exact pinning because major versions can have breaking changes which may effectively break the existing implementation. 
Besides I don't think this is an issue broadly as it was only reported by one person who may have a corrupted docker container.

---

Okay! Closed the PR.